src/dialog/ffc_editor.cpp

```diff
@@ -292,23 +312,60 @@ std::shared_ptr<GUI::Widget> FFCDialog::view()
 			TabPanel(
 				ptr = &ffctab,
 				TabRef(name = "Data", Row(
-					Rows<4>(
-						SWAPFIELD("X Pos:", ffc.x, SWAP_MIN, SWAP_MAX),
-						SWAPFIELD("X Speed:", ffc.dx, SWAP_MIN, SWAP_MAX),
+					Rows<6>(
+						SWAPFIELD("X Pos:", ffc.x, SWAP_MIN, SWAP_MAX, 
+							"This is the number of pixels from the left of the screen the FFC begins at.\n"
+							"Also note that if this value becomes smaller than - 32 or larger than 288,\n"
+							"it will either wrap around or be deactivated."),
+						SWAPFIELD("X Speed:", ffc.dx, SWAP_MIN, SWAP_MAX, 
+							"This is the initial velocity this FFC has parallel to the X-axis, in pixels per frame.\n"
+							"Positive numbers will move the FFC right, while a negative number can be entered to make\n"
+							"the FFC move left. Fractions may also be used.For example, entering a X Speed of 1 will make\n"
+							"the FFC move right 1 pixel every frame. Entering a X Speed of 0.5 will make the FFC move right\n"
+							"at 1 pixel per 2 frames"),
 						//
-						SWAPFIELD("Y Pos:", ffc.y, SWAP_MIN, SWAP_MAX),
-						SWAPFIELD("Y Speed:", ffc.dy, SWAP_MIN, SWAP_MAX),
+						SWAPFIELD("Y Pos:", ffc.y, SWAP_MIN, SWAP_MAX, 
+							"This is the number of pixels from the top of the screen the FFC begins at.\n"
+							"Note that if this value becomes smaller than -32 or larger than 208,\n"
+							"it will either wrap around or be deactivated."),
+						SWAPFIELD("Y Speed:", ffc.dy, SWAP_MIN, SWAP_MAX, 
+							"This is the initial velocity this FFC has parallel to the Y-axis, in pixels per frame.\n"
+							"Positive numbers will move the FFC down, while a negative number can be entered to make\n"
+							"the FFC move up. Fractions may also be used.For example, entering a Y Speed of 1 will make\n"
+							"the FFC move right 1 pixel every frame. Entering a Y Speed of 0.5 will make the FFC move down\n"
+							"at 1 pixel per 2 frames"),
 						//
-						SWAPFIELDB("Combo W:", ffc.fwid, 1, 64, 1),
-						SWAPFIELD("X Accel:", ffc.ax, SWAP_MIN, SWAP_MAX),
+						SWAPFIELDB("Combo W:", ffc.fwid, 1, 64, 1, 
+							"The Combo W. of a FFC denotes the width of the field of effect\n"
+							"that the FFC's Combo Type has in pixels. This also effects solidity and platforms!"),
+						SWAPFIELD("X Accel:", ffc.ax, SWAP_MIN, SWAP_MAX,
+							"This is the FFC's rightward acceleration.\n"
+							"A zero can be entered so that the FFC moves at a constant velocity,\n"
+							"or a negative number can be entered so the FFC accelerates leftward."),
```

Comment:
(This change would apply everywhere else too, not just this block; I'm only suggesting it for this one though)
```suggestion
							"This is the FFC's rightward acceleration."
							" A zero can be entered so that the FFC moves at a constant velocity,"
							" or a negative number can be entered so the FFC accelerates leftward."),
```
Generally, whitespace should go at the *start* of a line, rather than the end. Also, the info *should* be smart about breaking the lines for you if you just use spaces? (Depending on how it looks visually, leaving some manual line breaks for formatting is absolutely fine - and note that I'm only looking at this in github, not seeing how it looks in the editor, so definitely do what looks better; though I would be sure that the ` `s and `\n`s are at the *start* of the lines, regardless)